rescrape.py

```diff
@@ -1,31 +1,41 @@
-
-
 import configparser
 import geopy
 from geopy.geocoders import Nominatim
 from geopy.distance import geodesic
 import webbrowser
 import requests
 from bs4 import BeautifulSoup
-import time

```

Comment:
I tried running the code on 0.5 search radius (lots of http requests) with the 3 second sleep between each request commented out and it ran fine. I can't remember exactly what was causing my issues originally that made me put it in there. So we can take this out for right now.